pkg/odo/util/cmdutils.go

```diff
@@ -148,3 +148,9 @@ Additional help topics:{{range .Commands}}{{if .IsHelpCommand}}
 
 Use "{{.CommandPath}} [command] --help" for more information about a command.{{end}}
 `
+
+// ThrowContextError prints a context error if application/project is not found
+func ThrowContextError() error {
+	return errors.Errorf("Please specify the application name and project name" +
```

Comment:
You can just use:

```go
return errors.Errorf(`Please specify the application name and project name
Or use the command from inside a directory containing an odo component.`)
```

Reply:
The problem is it will look like this
```
// ThrowContextError prints a context error if application/project is not found
func ThrowContextError() error {
	return errors.Errorf(`Please specify the application name and project name
Or use the command from inside a directory containing an odo component.`)
}
```
It looks not properly formatted.